src/main/java/org/fogbowcloud/saps/engine/core/archiver/storage/AccessLink.java

```diff
@@ -0,0 +1,19 @@
+package org.fogbowcloud.saps.engine.core.archiver.storage;
+
+public class AccessLink {
+    private final String name;
+    private final String url;
+
+    public AccessLink(String name, String url) {
```

Comment:
name or url are alllowed to be null? or empty? if not, check and throw an IllegalArgumentException

Reply:
Done.